### PR TITLE
Feature/change password example

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes PXBLUE-2299 .

#### Changes proposed in this Pull Request:
- Add change password screen
- 
- 

#### To Test:
- http://localhost:4200/forms-and-validation/password

### CODE REVIEW

**Review by joebochill — COMMENTED** [review]
Spacing and sizing of the password criteria items looks off compared to what we have in the workflows:
image


Contents of the card should scroll, but not the title or the buttons

**Comment by huayunh** [comment]
May also want to upgrade @pxblue/angular-themes. The bottom line stroke looks different in this screenshot

File: src/app/pages/forms-and-validation/password/password-strength-check.component.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import * as Colors from '@pxblue/colors';
+
+@Component({
+    selector: 'app-password-strength-check',
+    template: `
+        <mat-list-item style="height: auto;" class="app-password-strength-check">
+            <mat-icon [style.color]="success ? Colors.blue[500] : Colors.gray[300]">{{ icon }}</mat-icon>
+            <span [style.color]="success ? Colors.gray[300] : Colors.black[500]" class="password-criteria">{{
+                validationMessage
+            }}</span>
```
Comment (joebochill) [inline src/app/pages/forms-and-validation/password/password-strength-check.component.ts:11]: we should use the mat-typography caption class here to make sure we have all of the correct typography styles (and then add our padding.margin on top of that)

**Review by joebochill — APPROVED** [review]
Code looks good here...the sizing and spacing comments I made before seem to be coming from two things:

1. Angular Material form fields automatically include space for error messages in their sizing, so if you specify the margin bottom at 32px, then if there is no error, the space will appear larger than that.
2. There is a discrepancy between what we have implemented for the criteria checkers in the auth workflows (subtitle2) and what is in the DSM (caption).

Not asking for any changes for either of those (at this time), but just wanted to point that out for informational purposes.

**Comment by joebochill** [comment]
@emclaug2 can you take a look at this today and see if anything sticks out to you?

**Comment by joebochill** [comment]
If there are no comments, I'm assuming this is good to merge in now.

**Review by jeffvg — APPROVED** [review]
looks good

**Review by emclaug2 — APPROVED** [review]

**Pull request merged**